### ISSUE TITLE
Accessibility: STARTTLS policy page should use descriptive link text

### ISSUE THREAD

**Issue by lenazun** [EFForg/starttls-frontend#186]
ref: https://redmine.eff.org/issues/17754

William T says:

The current text on the STARTTLS Policy List page should be improved for
users running screen readers. It currently reads:

"The list is hosted [here], and the corresponding signature is [here].

https://dl.eff.org/starttls-everywhere.policy.json
https://dl.eff.org/starttls-everywhere.policy.json.asc
[Here] is a detailed specification of the list’s format."

While sighted users can skim the page visually, blind and low vision
users often jump from link to link using the tab key, which a screen
reader would just read as "Link here" for the above links. This
creates a barrier for adoption with blind users. I haven't dug deep into
the rest of the site, but we should especially be taking accessibility
into account when launching projects intended for mass adoption and I've
noticed this on other EFF pages. Maybe we should add this to our
style guide and train staff on.

More info about phrasing links to be accessible for screen readers:
http://accessibility.oregonstate.edu//descriptivelinks
https://webaim.org/techniques/hypertext/link_text#text
